car/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/ipld/go-car/car
 go 1.13
 
 require (
-	github.com/ipld/go-car v0.1.0
-	github.com/urfave/cli v1.22.4
+	github.com/ipld/go-car v0.3.0
```

Comment:
bump go requirement to 1.15 while we're at it?

Reply:
sure, I actually removed go.sum and go.mod entirely (cause it seems like a reasonable way to "clean" things up) and recreated so manually put 1.13 back in there